app/views/layouts/application.html.erb

```diff
@@ -12,6 +12,8 @@
   </head>
 
   <body>
+    <p class="notice"><%= notice %></p>
```

Comment:
Was the navbar there when you made these changes?

Reply:
@SofieFiskerstrand No, I think you might have merged to master in the meantime, hence the conflict! Woops. I'll reintroduce the two lines of code and commit.